### ISSUE TITLE
Windows warning `Couldn't get modification time for registry key.` not fixed for footprint tests

### ISSUE THREAD

**Issue by verdx** [wazuh/wazuh#18938]
|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
| 4.6.0-alpha1 | wazuh-agent | Agent | Packages | Windows |

A warning has appeared in the `4.6.0-alpha1` Footprint tests which was previously reported as part of the issue https://github.com/wazuh/wazuh/issues/11088. This issue has since been closed as solved, and the rest of the errors mentioned do not appear, so it seems this error didn't really solve all of it.

The warning:
```
wazuh-agent WARNING: Couldn't get modification time for registry key.
```

The commentary in which it was mentioned in the closed issue is the following: https://github.com/wazuh/wazuh/issues/11088#issuecomment-1048776852

**Comment by nbertoldo** [comment]
It can be seen in the logs that this warning is displayed five times during the 2.5 day test. It seems to be related to the bugs fixed in #15986.

**Comment by nbertoldo** [comment]
### Conclusion
Considering the following:
- The test is run for 60 hrs.
- The FIM scan is done every 60 seconds.
- The ossec.conf contains 45 registry keys to be monitored.

Therefore, at least (one sub-key per registry key) 162000 access attempts are made to the `registry key modification time`. Considering that each registry key contains many sub-keys, the number increases considerably.
The logs show that the warning appears only 5 times during the test, i.e. a 0.003% chance of occurrence. In addition, it is a `warning` message, not an `error`. 
So we can conclude that the incidence of this warning is not significant.
